File: src/utils/people_power.ts
for (let number = 1; number <= 100; number++) {
  let result = [];
  if (number % 3 === 0) {
    result.push("People");
  }
  if (number % 5 === 0) {
    result.push("Power");
  }
  console.log(result.length > 0 ? result.join(" ") : number.toString());
}

File: src/utils/people_power.rb
(1..100).each { |number|
 list = []
 
 if number % 3 == 0
  list.push("People")
 end
 if number % 5 == 0
  list.push("Power")
 end

 puts list.size > 0 ? list.join(" ") : number.to_s
}